src/index.js

```diff
@@ -310,7 +310,7 @@ function createConfig(options, entry, format, writeMeta) {
 			strict: options.strict===true,
 			legacy: true,
 			freeze: false,
-			sourcemap: true,
+			sourcemap: options.sourcemap===true || options.sourcemap===undefined,
```

Comment:
This can just be `options.sourcemap` since the default is set to `true` 😄

Reply:
Well that was my assumption as well, at first 🙂. Seems that the cli would have it defined from sade as true, but running it through the node package, would end up undefinded.

Did I get it right?

Reply:
I think it's because your first attempt was using `sourceMap` -- I just checked out your current branch it's working as expected. There's no need for any `===` checks 👍

Reply:
Ha, seems like @developit did some trickery check there to cover both cases, for default `true` options 😄just checking it that it's not false `options.sourcemap!==false`. That covers both cases, cli and node.

Node it's different, the options object it's not validated by `sade` just a plain config option.

Reply:
Right about here https://github.com/developit/microbundle/blob/master/src/index.js#L265 I was looking how the other always true values are behaving. And here's how the options are passed to `microbundle` function https://github.com/developit/microbundle/blob/master/test/index.test.js#L29.

Reply:
Those existed for Yargs, aka before the switch to Sade 😄 I didn't bother converting them to make the PR diff easier to follow.

Sade uses [`mri`](https://github.com/lukeed/mri) under the hood, which will _always_ ensure that a flag value will be `boolean|string|number` if specified or if an initial value is provided.

Reply:
Ha, nice! That makes sense. Also, kudos on the `mri` 😃that's impressive perf, you got there.

Reply:
Haha, thanks 🙃 Flag parsing perf doesn't matter too much for the majority of cases, but is nice to have. The simple utilities of type-casting & assumed booleans are more useful... allowing you avoid all these `!==` manual checks in your apps.